packages/curve-ui-kit/src/features/connect-wallet/lib/wagmi/wagmi-config.ts

```diff
@@ -31,31 +31,37 @@ export const createWagmiConfig = memoize(
   <ChainId extends number, NetworkConfig extends BaseConfig>(networks: Record<ChainId, NetworkConfig>) => {
     const chains = Object.fromEntries(wagmiChains.map((chain) => [chain.id, chain]))
     const networkEntries = Object.entries(networks).map(([id, config]) => [+id, config]) as [ChainId, NetworkConfig][]
+
+    /**
+     * Gets a list of the RPC URLs for a given chain.
+     * First the hardcoded RPC URLs, then the CurveJS URL and then all default wagmi URLs
+     */
+    const getRpcUrls = (id: ChainId, config: NetworkConfig, wagmiChain?: Chain) =>
+      uniq([...(RPC[id] ?? []), config.rpcUrl, ...(wagmiChain?.rpcUrls.default.http ?? [])])
```

Comment:
Just be aware that not using `undefined` means chains without configured RPCs work. `undefined` means to use wagmi's RPC, which I believe for Ethereum now is merkle.io.

Maybe add a fallback like `RPC[id] ?? [undefined]`, so `undefined` is used in case of no RPCs.